index.js

```diff
@@ -31,12 +31,15 @@ const runBsbSync = () => {
   execFileSync(bsb, ['-make-world'], { stdio: 'pipe' })
 }
 
-const getBsbErrorMessages = err =>
-  err.match(/File [\s\S]*?:\nError: [\s\S]*?(?=ninja|\n\n)/g)
+const getBsbErrorMessages = err => {
+  const normalErr = /File [\s\S]*?:\nError: [\s\S]*?(?=ninja|\n\n)/g
+  const fatalErr = () => /Fatal error:/g
```

Comment:
*lazy* here, will not evaluate if `normalErr` is true. Should gain a little performance here I think.